ArduCopter/config.h

```diff
@@ -705,3 +705,7 @@
 #ifndef HAL_FRAME_TYPE_DEFAULT
 #define HAL_FRAME_TYPE_DEFAULT AP_Motors::MOTOR_FRAME_TYPE_X
 #endif
+
+#if CUSTOMCONTROL_ENABLED == ENABLED && FRAME_CONFIG == HELI_FRAME
+  #error Custom controller requires MULTI_FRAME
```

Comment:
Looks like this warning is catching us out now we have enabled on SITL. Any reason it would not work on heli in SITL? Maybe we just skip this check for SITL?

Reply:
I had a problem while passing base class type into the custom controller and its backends, so for simplicity, I only focused on multicopter. After looking at it again, we can now compile on Heli, as well.